2021-11-29 16:19:14,580 - trainer.py[line:25] - INFO: Running with identifier 20211129_161914
2021-11-29 16:19:14,580 - trainer.py[line:27] - INFO: Saving log to log\cifar10\20211129_161914\run.log
2021-11-29 16:19:14,580 - trainer.py[line:28] - INFO: Using args: [('log_name', 'cifar10'), ('data_train', 'dataset/cifar_10_train'), ('data_eval', 'dataset/cifar_10_test'), ('output_checkpoint_dir', 'checkpoint'), ('eval_per_epoch', 1), ('log_dir', 'log'), ('tf_dir', 'tensorboard'), ('output_dir', 'output'), ('not_ignore_warning', False), ('enable_tensorboard', True)]
2021-11-29 16:19:14,582 - trainer.py[line:39] - INFO: Using tensorboard, store at: tensorboard\cifar10\20211129_161914
2021-11-29 16:19:14,582 - train.py[line:97] - INFO: Running model resnet50
2021-11-29 16:20:12,928 - trainer.py[line:53] - INFO: Keyboard Interrupt
